app/src/main/java/com/jerboa/ui/components/drawer/Drawer.kt

```diff
@@ -439,15 +442,17 @@ fun AvatarAndAccountName(
             }
         }
         Column {
-            PersonName(
-                name = myPerson?.display_name ?: account.name,
-                color = MaterialTheme.colorScheme.onSurface,
+            ItemAndInstanceTitle(
+                title = myPerson?.getDisplayName() ?: account.name,
+                actorId = myPerson?.actor_id,
+                local = true,
+                itemColor = MaterialTheme.colorScheme.onSurface,
             )
             Text(
                 text = if (account.isAnon()) "" else "${account.name}@${account.instance}",
```

Comment:
I think I am missing something but isn't this twice the same? The `ItemAndInstanceTitle` above it aims to achieve this text right?

Reply:
The top one shows your display name, while the bottom shows your account name. I'll clean this up tho.